azure-pipelines.yml

```diff
@@ -65,10 +55,10 @@ stages:
           condition: and(succeeded(), eq(variables['installDependencies'], 'true'))
 
         - bash: |
-            ./build.sh --ci --restore --build --pack -c $(_BuildConfig)
```

Comment:
I think the `--ci` is important so it generates the correct logs on AzDO.

Reply:
Urgh, I think the `--ci` is also why we're getting a `-ci` suffix on the nupkg version. I'll re-add it, but the version number might be a mess until we figure this out